### ISSUE TITLE
Batch_normalization

### ISSUE THREAD

**Issue by gcroci2** [DeepRank/3D-Vac#131]
Batch normalization is a feature that we add between the layers of the neural network and it continuously takes the output from the previous layer and normalizes it before sending it to the next layer. This has the effect of stabilizing the neural network. Batch normalization is also used to maintain the distribution of the data.

This problem is framed as an internal covariate shift. To maintain the similar distribution of data we use batch normalization by normalizing the outputs using mean=0, standard dev=1 (μ=0,σ=1). By using this technique, the model is trained faster and it also increases the accuracy of the model compared to a model that does not use the batch normalization [More info [here](https://analyticsindiamag.com/hands-on-guide-to-implement-batch-normalization-in-deep-learning-models/)].

![image](https://user-images.githubusercontent.com/55382553/223752718-e43a9528-2569-4b7f-a70d-f441701028ad.png)

Add batch normalization to the network, and compare:
- [x] standardization only (see #124, exp with standardization), lr=0.001
- [x] standardization + batch normalization, lr=0.001
  - batch norm inserted before Relu activation function, see the architecture below (method 1)
- [x] batch normalization only (maybe standardization is not needed), lr=0.001
- [x] standardization + batch normalization, lr=0.01 to see if decreasing lr helps batch normalization
  - batch norm inserted before Relu activation function, see the architecture below (method 1)
- [x] standardization + batch normalization, lr=0.01
  - [x] batch norm inserted after Relu activation function, see the architecture below (method 2)
  - [x]  batch norm inserted between the two convolutional layers, see the architecture below (method 3)
  - [x]  batch norm inserted between the two convolutional layers and after, using Relu activation function as well (method 4)
- We did try out [GraphNorm](https://github.com/lsj2408/GraphNorm), but stopped the experiment due the different geometric package we are using compared to theirs. We are using pytorch geomtric and the original graph norm is using dgl.
